site/public/js/forum.js

```diff
@@ -95,7 +95,7 @@ function uploadImageAttachments(attachment_box) {
         const filename = target.attr('fname');
         // eslint-disable-next-line no-undef
         for (let j = 0; j < file_array[part-1].length; j++) {
-            // eslint-disable-next-line no-undef
+            // eslint-disable-next-line no-undef, eqeqeq
             if (file_array[part-1][j].name == filename) {
```

Comment:
I was able to get the same result when uploading an attachment to a forum post after replacing `file_array[part-1][j].name == filename` with `file_array[part-1][j].name === filename`.  When getting the type of `file_array[part-1][j].name` and `filename`, `string` is returned for both.